closed/src/java.base/share/classes/sun/net/util/AIX.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
+ * ===========================================================================
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * IBM designates this particular file as subject to the "Classpath" exception
+ * as provided by IBM in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * ===========================================================================
+ */
+
+package sun.net.util;
+
+public final class AIX {
+   // Flag indicating whether the operating system is AIX.
+   public static final boolean isAIX = "AIX".equals(GetPropertyAction.privilegedGetProperty("os.name"));
```

Comment:
Please delete blank line 24 and indent four spaces per level (like the rest of this repo), not three.
This is missing:
```java
import sun.security.action.GetPropertyAction;
```